### ISSUE TITLE
Perspective drawing helper

### ISSUE THREAD

**Issue by dacap** [aseprite/aseprite#1496]
New tool helper to draw lines on perspective:

1. Definition of the horizon line (show line during definition)
2. Definition of vanishing points on the horizon / center lines
3. using mouse and or keyboard shortcuts to select one/multiple vanishing points
4. then by holding the left mouse button it will create lines pointing to the vanishing points starting from the point where the mouse button was pressed down. Moving the mouse left and right during holding will increase/decrease the lenght of the lines. (a preview of the lines will be shown during this phase)
5. with the release of the left mouse button the line will be drawn onto the canvas

the tools options should include the original line tool's options.

(feature requested via email at support@aseprite.org)

**Comment by juanpaexpedite** [comment]
Yes it is very interesting, I'm beginning to create rooms in Aseprite and would be great adding that vanishing points to make lines from there. Also being able to create (floor) rectangles in perspective will make things even faster.

**Comment by domdere** [comment]
yeah this would be seriously awesome, right now when i do this i have to do it in a separate app (for general illustrations, not pixel art specific) and import the guidelines into aseprite and clean up the pixels etc...

I'd probably find it handy for lighting etc too

**Comment by muchitto** [comment]
This would be super helpful. Especially for drawing rooms and backgrounds!

**Comment by dacap** [comment]
Maybe related with #481
